scripts/ipfs-pi-stream/process-stream.sh

```diff
@@ -14,93 +14,98 @@ function startFFmpeg() {
   while true; do
     mv ~/ffmpeg.log ~/ffmpeg.1
     echo 1 > ~/stream-reset
-    ffmpeg -f video4linux2 -input_format h264 -video_size 1280x720 -framerate 30 -i /dev/video0 -vcodec copy -hls_time ${HLS_TIME}  ${what}.m3u8   > ~/ffmpeg 2>&1
+    ffmpeg -f video4linux2 -input_format h264 -video_size 1280x720 -framerate 30 -i /dev/video0 -vcodec copy -hls_time "${HLS_TIME}" "${what}.m3u8"   > ~/ffmpeg 2>&1
     sleep 0.5
   done
 }
 
 # Create directory for HLS content
 
 currentpath="$HOME/live"
-sudo umount $currentpath
-rm -rf $currentpath
-mkdir $currentpath
-sudo mount -t tmpfs tmpfs $currentpath
-cd $currentpath
+sudo umount "${currentpath}"
+rm -rf "${currentpath}"
+mkdir "${currentpath}"
+sudo mount -t tmpfs tmpfs "${currentpath}"
+# shellcheck disable=SC2164
+cd "${currentpath}"
 
 what="$(date +%Y%m%d%H%M)-LIVE"
 
 # Start ffmpeg in background
 startFFmpeg &
 
 while true; do
-  nextfile=$(ls -tr ${what}*.ts 2>/dev/null | tail -n 1)
+#TODO# Fix this one
```

Comment:
Probably best to refactor this script along with IPFS-Live-Stream to get them a bit more inline. We know what will be returned by this ls so lets leave it as is for now.

Reply:
I don't understand what you mean

Reply:
shellcheck states that LS is unpredictable.  In this case ls is predictable because we know what it is going to be. 

this script has gone through quite a few changes vs that IPFS-LIVE_STREAM version, id like to bring them to be aligned a little more so im ignoring this issue for now